streamz/core.py

```diff
@@ -991,6 +1013,27 @@ def __init__(self, *upstreams, **kwargs):
             self.emit_on = upstreams
         Stream.__init__(self, upstreams=upstreams, **kwargs)
 
+    def _add_upstream(self, upstream):
+        self.last.append(None)
```

Comment:
Again, I'd appreciate a comment for why an override is necessary here.

Have you now covered all classes which have multiple input streams?

Reply:
I haven't handled `zip_latest` yet.

Reply:
Is `zip_latest` out of scope for this PR, would you say?

Reply:
Yeah, I don't use it that much (despite creating it). It turned out that it had a whole bunch of state storage issues that should most likely be managed first.